src/components/WalletProvider/WalletProvider.tsx

```diff
@@ -0,0 +1,80 @@
+import {
+    StellarWalletsKit,
+    WalletNetwork,
+    allowAllModules,
+    LOBSTR_ID,
+  } from '@creit.tech/stellar-wallets-kit';
+import { Layout } from '@stellar/design-system';
+import { useEffect, useMemo, useState } from 'react';
+
+export const WalletProvider = () => {
+    const [address, setAddress] = useState<string | null>(null);
+
+
+    const kit = useMemo(() => {
+      return new StellarWalletsKit({
+        network: WalletNetwork.TESTNET,
+        selectedWalletId: LOBSTR_ID,
```

Comment:
We should save which wallet the user selected in localStorage (probably; feel free to suggest some other way to store it, like a cookie) and load that wallet instead of always picking lobstr